dbt_subprojects/daily_spellbook/models/_projects/lifi/arbitrum/lifi_arbitrum_transfers.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(
+    schema = 'lifi_arbitrum',
+    alias = 'transfers',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['transfer_id'],
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+with source_data as (
+    {{ lifi_extract_bridge_data('arbitrum') }}
+),
+
+tokens_mapped as (
+    select
+        *,
+        case
+            when sendingAssetId = '0x0000000000000000000000000000000000000000'
+            then '0x82af49447d8a07e3bd95bd0d56f35241523fbab1' -- WETH
+            when sendingAssetId = '0x3405a1bd46b85c5c029483fbecf2f3e611026e45'
+            then '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8' -- USDC
+            else sendingAssetId
```

Comment:
i'm fairly certain your latest failure is around these addresses having single quotes. can you remove, then should default to varbinary

Reply:
you catched it while i slept after pushing the changes, thanks!